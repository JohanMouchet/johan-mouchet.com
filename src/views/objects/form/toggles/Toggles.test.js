import React from "react";
import { shallow } from "enzyme";
import { Checkbox, Radio } from ".";

it("Renders <Checkbox /> without crashing", () => {
  shallow(<Checkbox />);
});

it("Renders <Radio /> without crashing", () => {
  shallow(<Radio />);
});
