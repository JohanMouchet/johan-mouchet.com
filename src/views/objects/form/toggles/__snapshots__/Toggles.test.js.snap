// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Checkbox /> without crashing 1`] = `
<label
  className="o-form__toggle o-form__toggle--checkbox"
>
  <input
    className="o-form__toggle-input"
    onChange={[Function]}
    type="checkbox"
  />
  <span
    className="o-form__toggle-icon"
  />
</label>
`;

exports[`Renders <Radio /> without crashing 1`] = `
<label
  className="o-form__toggle o-form__toggle--radio"
>
  <input
    className="o-form__toggle-input"
    onChange={[Function]}
    type="radio"
  />
  <span
    className="o-form__toggle-icon"
  />
</label>
`;
